scripts/jenkins/groovy/buildH2O3.groovy

```diff
@@ -1,6 +1,6 @@
 def call(final pipelineContext) {
 
-    final String PYTHON_VERSION = '3.6'
+    final String PYTHON_VERSION = '3.7'
```

Comment:
What does it mean? We won't run python 3.6 tests anymore?

Reply:
No, I build python 3.6 from source (https://github.com/h2oai/h2o-3/pull/15964/files#diff-91b50511ed257ab19e2d065fbe2ad897f21af2cb2131f3a87b0f503de1b162efR37-R55).

My understanding is that this setting influences only the build environment for h2o-3. 

We create the docker build-base image from harbor.h2o.ai/opsh2oai/h2o-3/dev-r-3.4.1 which is build on dev-r-base which is based on harbor.h2o.ai/opsh2oai/h2o-3/dev-python-3.7 image. This was a state before my changes. But it wasn't buildable since python 3.6 was missing.

I think I could revert this particular change without much trouble but I'd have to use python 3.6 in R images and since the python 3.6 is not supported in ubuntu 22.04, I think it's better to have supported python versions in our test images by default.

Reply:
> No, I build python 3.6 from source 

Great, thanks  you!